tests/core/test_simd_float32x4.c

```diff
@@ -5,9 +5,25 @@
 #include <math.h>
 #include <assert.h>
 
+void printFloat(float n) {
```

Comment:
Do you know what is the root cause of the divergence? Looks ok to have this then, but I'm surprised that this even differs?